Lesson2/SecondHomework.playground/Contents.swift

```diff
@@ -32,15 +45,21 @@ class StackStatistics: Stack {
         super.push(element: element)
     }
     
-    override func pop() -> Int? {
+    /// Removes and returns the top element of the stack. Returns nil when the invoking stack is empty.
+    /// - Complexity: O(1)
+    /// - Returns: The top element of the stack or nil.
+    override func pop() -> T? {
         if !minimalValues.isEmpty {
             minimalValues.removeLast()
         }
         return super.pop()
         
     }
     
-    func minimumElement() -> Int? {
+    /// Returns the last minimum element in the stack. Returns nil when the invoking stack is empty.
```

Comment:
```suggestion
    /// Returns the minimum element in the stack. Returns nil when the invoking stack is empty.
```
what does that mean "the last minimum element"? It's confusing a bit.

Reply:
Thanks. Corrected the description